packages/world/src/WorldResourceId.sol

```diff
@@ -10,7 +10,7 @@ import { RESOURCE_NAMESPACE, MASK_RESOURCE_NAMESPACE } from "./worldResourceType
 uint256 constant NAMESPACE_BITS = 14 * 8;
 uint256 constant NAME_BITS = 16 * 8;
 
-bytes16 constant ROOT_NAMESPACE_STRING = bytes16("ROOT_NAMESPACE");
+bytes14 constant ROOT_NAMESPACE_STRING = bytes14("ROOT_NAMESPACE");
 bytes16 constant ROOT_NAME_STRING = bytes16("ROOT_NAME");
```

Comment:
@alvrs A few notes/questions for us:
- Since these strings are just for debugging, they don't need to be type-aligned with the actual namespace/name types, right? They could technically be any length?
- I have been using an empty string in ~all of the client code, and wonder if we should align on that here. Alternatively, I wonder if we should use a less likely string like `<root>` or something.
- Should we inline this into `toString` (since that's the only place it's used) instead of defining up here, so we don't need to specify the type?
- Should we use `toTrimmedString` for namespace and name inside `toString`?

Reply:
> Since these strings are just for debugging, they don't need to be type-aligned with the actual namespace/name types, right? They could technically be any length?

Correct (but kinda agree using something else than `bytes14` can lead to confusion)

> I have been using an empty string for root namespace/name in ~all of the client code, and wonder if we should align on that here. Alternatively, I wonder if we should use a less likely string like <root> or something.

If we do that we don't even need the ternary operator here anymore since the actual namespace/name is already empty. The original reason why we replaced the empty string here with a more explicit one was that `ns::` might be less easy to understand than `ns:ROOT_NAMESPACE:ROOT_NAME`. (No strong opinion on which string we use, also happy to make it `ns:<root>:<root>`)

> Should we inline this into toString (since that's the only place it's used) instead of defining up here, so we don't need to specify the type?

Yeah that makes sense! (still need to specify the type tho because the ternary operator is unhappy if the result is `string | bytes14`)

> Should we use toTrimmedString for namespace and name inside toString?

We could, but if it's logged as string the `0`'s are already not displayed, and if it's logged as raw bytes it might be a tiny little bit easier to figure out what the log means if it always has the same length (34 bytes). On the other hand advantage of using `toTrimmedString` would mean it's possible to compare the result with an inline string, which also might be useful.

Reply:
I like it!

Given that this method is meant to be for debug/human-readability purposes and comparison/parsing should be done against the resource ID itself, I am keen for this to be more string like and use `toTrimmedString` around the whole thing. But would be curious to see gas differences before making the final call.